### PR TITLE
Update AssemblyScript syntax

### PR DESCRIPTION
**Pull request by bowenwang1996**
Update assemblyscript syntax after the runtime change. Requires https://github.com/nearprotocol/assemblyscript/pull/36, https://github.com/nearprotocol/near-runtime-ts/pull/27, and https://github.com/nearprotocol/assemblyscript-json/pull/13.

### CODE REVIEW

File: blank_project/assembly/main.ts [hunk 1]
```diff
@@ -12,7 +9,7 @@ import { Greeter } from "./model.near";
 // using `export` keyword.
 
 export function hello(): string {
-  let greeter = new Greeter("Hello");
+  let greeter = new Greeter("hello");
```
Comment (vgrichina) [inline blank_project/assembly/main.ts]: why is this changed?

Comment (bowenwang1996) [reply blank_project/assembly/main.ts]: Not sure. I'll change it back
